src/server/index.test.js

```diff
@@ -67,19 +68,21 @@ test('sync', () => {
   // Sync causes the server to respond.
   expect(io.socket.emit).toHaveBeenCalledTimes(0);
   io.socket.receive('sync', 'gameID');
-  expect(io.socket.emit).toHaveBeenCalledTimes(1);
+
+  expect(io.socket.emit).toHaveBeenCalledTimes(1)
   expect(spy).toHaveBeenCalled();
 
   // Sync a second time does not create a game.
   spy.mockReset();
   io.socket.receive('sync', 'gameID');
+
   expect(io.socket.emit).toHaveBeenCalledTimes(2);
   expect(spy).not.toHaveBeenCalled();
 
   spy.mockRestore();
 });
 
-test('action', () => {
+test('action', async () => {
```

Comment:
async not needed, because not await exist

Reply:
Agree, but I forgot to remove it after I clean `await` s when it didn't work